inst/CIDatabase/travis-tool.sh

```diff
@@ -292,6 +292,9 @@ RunTests() {
     fi
     _R_CHECK_CRAN_INCOMING_=${_R_CHECK_CRAN_INCOMING_} R_QPDF=true R CMD check "${FILE}" ${R_CHECK_ARGS} ${R_CHECK_INSTALL_ARGS}
 
+    # Check examples flagged with \donttest
+    check()
+    
```

Comment:
I added that run_examples in. 

I agree that we should change the name to appveyor-tool.sh